### ISSUE TITLE
 QEditor definitions prop type

### ISSUE THREAD

**Issue by cirolosapio** [quasarframework/quasar#14141]
### What happened?

QEditorProps interface is defined as: 
```ts
export interface QEditorProps {
  ...
  definitions?:
    | {
        /**
         * Label of the button
         */
        label?: string;
        /**
         * Text to be displayed as a tooltip on hover
         */
        tip?: string
  ...
```

### What did you expect to happen?

the `definitions` prop should be `Record<string, { ... }>`

### Reproduction URL

https://stackblitz.com/edit/vitejs-vite-po3wz7?file=src%2Fcomponents%2FTestComp.ts

### How to reproduce?

1. Go to the provided reproduction link
2. See type error on definitions computed

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar), TypeScript Support

**Comment by yusufkandemir** [comment]
Thank you for reporting this. This is probably the last one of the type errors. It was annoying me for a while, but couldn't find the right time and kept forgetting to open a ticket or fix it.

**Comment by cirolosapio** [comment]
i want to contribute but can't find those files
can you please tell me where are they?

**Comment by yusufkandemir** [comment]
@heartbeatLV 
https://github.com/quasarframework/quasar/blob/efdff8b2191032fce93319e303a794b1478b3fe8/ui/src/components/editor/QEditor.json#L72-L150

**Comment by rstoenescu** [comment]
Enhancement will be available in Quasar 2.8.0
